querybook/server/models/comment.py

```diff
@@ -0,0 +1,84 @@
+import sqlalchemy as sql
+from app import db
+from lib.sqlalchemy import CRUDMixin
+from sqlalchemy.orm import relationship
+from const.db import mediumtext_length, name_length, now
+
+Base = db.Base
+
+
+class Comment(CRUDMixin, Base):
+    __tablename__ = "comment"
+
+    id = sql.Column(sql.Integer, primary_key=True)
+
+    created_at = sql.Column(sql.DateTime, default=now)
+    updated_at = sql.Column(sql.DateTime, default=now)
+    created_by = sql.Column(
+        sql.Integer, sql.ForeignKey("user.id", ondelete="SET NULL"), nullable=True
+    )
+
+    text = sql.Column(sql.Text(length=mediumtext_length))
+
+    parent_comment_id = sql.Column(
+        sql.Integer,
+        sql.ForeignKey("comment.id", ondelete="CASCADE"),
+        index=True,
```

Comment:
why are we indexing on foreign keys? https://stackoverflow.com/questions/304317/does-mysql-index-foreign-key-columns-automatically